src/shapes/text.class.js

```diff
@@ -175,8 +175,8 @@
      * @default
      */
     superscript: {
-      size:      0.60, // fontSize factor
-      baseline:  -0.67  // baseline-shift factor (upwards)
+      size:      0.83, // fontSize factor
+      baseline: -0.36  // baseline-shift factor (upwards)
```

Comment:
the source you documented:
![image](https://user-images.githubusercontent.com/1194048/29373521-d52cc726-82ae-11e7-8333-f60c462ce352.png)

has references to behaviours of other apps.